testdata/baselines/reference/submoduleAccepted/conformance/checkJsdocTypeTag6.errors.txt.diff

```diff
@@ -2,46 +2,60 @@
- test.js(27,7): error TS2322: Type '(more: any) => void' is not assignable to type '() => void'.
-   Target signature provides too few arguments. Expected 1 or more, but got 0.
- test.js(30,7): error TS2322: Type '(more: any) => void' is not assignable to type '() => void'.
-   Target signature provides too few arguments. Expected 1 or more, but got 0.
+-test.js(27,7): error TS2322: Type '(more: any) => void' is not assignable to type '() => void'.
+-  Target signature provides too few arguments. Expected 1 or more, but got 0.
+-test.js(30,7): error TS2322: Type '(more: any) => void' is not assignable to type '() => void'.
+-  Target signature provides too few arguments. Expected 1 or more, but got 0.
 -test.js(34,3): error TS2322: Type '(more: any) => void' is not assignable to type '() => void'.
 -  Target signature provides too few arguments. Expected 1 or more, but got 0.
 -
 -
 -==== test.js (7 errors) ====
++test.js(10,12): error TS8030: A JSDoc '@type' tag on a function must have a signature with the correct number of arguments.
++test.js(23,12): error TS8030: A JSDoc '@type' tag on a function must have a signature with the correct number of arguments.
 +
 +
-+==== test.js (3 errors) ====
++==== test.js (4 errors) ====
      /** @type {number} */
--               ~~~~~~
+                ~~~~~~
 -!!! error TS8030: The type of a function declaration must match the function's signature.
++!!! error TS8030: A JSDoc '@type' tag on a function must have a signature with the correct number of arguments.
      function f() {
          return 1
      }
-@@= skipped -24, +17 lines =@@
-     }
+@@= skipped -25, +19 lines =@@
      
      /** @type {(a: number) => number} */
--               ~~~~~~~~~~~~~~~~~~~~~
+                ~~~~~~~~~~~~~~~~~~~~~
 -!!! error TS8030: The type of a function declaration must match the function's signature.
++!!! error TS8030: A JSDoc '@type' tag on a function must have a signature with the correct number of arguments.
      function add1(a, b) { return a + b; }
      
      /** @type {(a: number, b: number) => number} */
-@@= skipped -15, +13 lines =@@
-     // They can't have more parameters than the type/context.
+@@= skipped -15, +15 lines =@@
      
      /** @type {() => void} */
--               ~~~~~~~~~~
+                ~~~~~~~~~~
 -!!! error TS8030: The type of a function declaration must match the function's signature.
++!!! error TS8030: A JSDoc '@type' tag on a function must have a signature with the correct number of arguments.
      function funcWithMoreParameters(more) {} // error
      
      /** @type {() => void} */
-@@= skipped -19, +17 lines =@@
+     const variableWithMoreParameters = function (more) {}; // error
+-          ~~~~~~~~~~~~~~~~~~~~~~~~~~
+-!!! error TS2322: Type '(more: any) => void' is not assignable to type '() => void'.
+-!!! error TS2322:   Target signature provides too few arguments. Expected 1 or more, but got 0.
```

Comment:
Hm, are we missing some checking here?

Reply:
yes, fixed now, although it's not contextually typed anymore so it gets the less-informative new error.